touchforms/formplayer/migrations/0003_auto__add_session.py

```diff
@@ -0,0 +1,90 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        
+        # Adding model 'Session'
+        db.create_table(u'formplayer_session', (
```

Comment:
Just was thinking about this more. We currently have a cron job that cleans the touchform files so it doesn't get out of hand. I can see this table growing out of control. We should think about adding a trigger that will delete the entry after a certain amount of time. especially cause the row will actually be quite big (contains the instance xml)